lib/sinatra/config_file.rb

```diff
@@ -128,9 +128,9 @@ def config_file(*paths)
       Dir.chdir(root || '.') do
         paths.each do |pattern|
           Dir.glob(pattern) do |file|
+            raise 'Invalid config file: Only .yml and .erb extensions are supported' unless ['.yml', '.erb'].include?(File.extname(file))
```

Comment:
I don't want to raise a plain Exception here, can we introduce a new exception class specific to this module?